dvc/command/metrics.py

```diff
@@ -12,7 +12,32 @@ def _show(self, metrics):
                 logger.info("{}:".format(branch))
 
             for fname, metric in val.items():
-                logger.info("\t{}: {}".format(fname, metric))
+                self._output_formated_metric(fname, metric)
+
+    def _output_formated_metric(self, fname, metric):
+        content, typ = metric
+
+        delimiter = {"csv": ",", "hcsv": ",", "tsv": "\t", "htsv": "\t"}.get(
+            typ
+        )
+
+        if not delimiter or "\n" not in content:
```

Comment:
in what cases do we need this `if`? (I know that sometimes content is actually should be a list, are there actual content (single line, or without delimiters) that could not be processed with code below (may be fixed a bit))?

Reply:
Because we don't want to process `json`s nor `raw` files.

Also, if the output is a one-liner, it could fit on the same line to be consistent with `raw`:
```
metrics.csv: 0.36,0.88
```